### ISSUE TITLE
How to resume training from the last model_epoch.pt

### ISSUE THREAD

**Issue by atrah22** [lmbxmu/FilterSketch#1]
Hello,
I am trying to resume the training from epoch=19 as the last saved model is model_19.pt.
However, the training accuracy significantly drops compared to the actual accuracy of model_19.pt
I am pruning resnet50v1 model.

I am using remote server and my submitted jobs terminates in 24 hours. I hence need to start the job again from the previously trained model.

BRs,
atul

**Comment by lmbxmu** [comment]
Truly apologize. I haven't notice this since I did not receive any email towards this issue.

Can your provide me more about your training loggers and configurations ?

Besides, my WeChat (lmbcxy) is suggested to be added (if you are using it) so that we can have more convenient communications.

**Comment by atrah22** [comment]
Hello,
I ended up not pursuing this project anymore. I reproduced ABCPruner github project (https://github.com/lmbxmu/abcpruner) and had no problem running the code. Is ABC Pruner better than FilterSketch in terms of accuracy?

**Comment by lmbxmu** [comment]
Yes. But the results are not deterministic since the structure search is not fixed.

**Comment by lmbxmu** [comment]
As I can remember, I did not provide the resuming function in this project. I guess that you finish it by yourself and something is wrong in your codes. If necessary, let me know and I can do this for you.
